CHANGES.rst

```diff
@@ -4,6 +4,18 @@
 Service fixes and enhancements
 ------------------------------
 
+gemini
+^^^^^^
+
+- Renamed and deprecated Observations and ObservationsClass in GEMINI module. [#1885]
```

Comment:
this is nitpicking, so I only point out to show better practice: changelog should point out what has changed, and what the new solution is. No need to mention the module name in the changelog as the section above already contains that information. The backticks are to highlight the class names when rendered for the documentation.
```suggestion
- Renamed and deprecated ``Observations`` and ``ObservationsClass`` to ``GeminiObservations`` and ``GeminiObservationsClass`` . [#1885]
```